src/libgap-api.h

```diff
@@ -179,6 +179,11 @@ Obj GAP_EvalString(const char * cmd);
 // defined.
 Obj GAP_ValueGlobalVariable(const char * name);
 
+// Combines gVarName and AssGVar. For a given <name> and <value>,
```

Comment:
Please remove that first sentence. Explaining an API in terms of other APIs which are not even part of the libgap API is harmful.